src/m365/spo/commands/list/list-add.ts

```diff
@@ -762,9 +760,7 @@ class SpoListAddCommand extends SpoCommand {
       requestBody.IsApplicationList = options.isApplicationList;
     }
 
-    if (options.listExperienceOptions) {
-      requestBody.ListExperienceOptions = options.listExperienceOptions;
-    }
+    requestBody.ListExperienceOptions = options.listExperienceOptions ? ListExperience[(options.listExperienceOptions.trim() as keyof typeof ListExperience)] : ListExperience.Auto;
```

Comment:
Since we removed the default value in the docs (which is good in my opinion), we shouldn't also use a default value here. Let's just set the value when the option is set.